clang-tidy-checks/setup.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+function info() {
+  echo "[info]: " $1
```

Comment:
could you quote this? https://github.com/koalaman/shellcheck/wiki/SC2086
```suggestion
  echo "[info]: " "$1"
```